src/pages/ProfilePage.js

```diff
@@ -0,0 +1,237 @@
+                            </AttachmentModal>
+                            {Boolean(displayName) && (
+                                <Text
+                                    style={[styles.textHeadline, styles.mb6, styles.pre]}
+                                    numberOfLines={1}
+                                >
+                                    {displayName}
+                                </Text>
+                            )}
+                            {login ? (
+                                <View style={[styles.mb6, styles.detailsPageSectionContainer, styles.w100]}>
+                                    <Text
+                                        style={[styles.textLabelSupporting, styles.mb1]}
+                                        numberOfLines={1}
+                                    >
+                                        {props.translate(isSMSLogin ? 'common.phoneNumber' : 'common.email')}
+                                    </Text>
+                                    <CommunicationsLink value={phoneOrEmail}>
+                                        <Tooltip text={phoneOrEmail}>
+                                            <Text numberOfLines={1}>{isSMSLogin ? props.formatPhoneNumber(phoneNumber) : login}</Text>
+                                        </Tooltip>
+                                    </CommunicationsLink>
+                                </View>
+                            ) : null}
+                            {pronouns ? (
+                                <View style={[styles.mb6, styles.detailsPageSectionContainer]}>
+                                    <Text
+                                        style={[styles.textLabelSupporting, styles.mb1]}
+                                        numberOfLines={1}
+                                    >
+                                        {props.translate('profilePage.preferredPronouns')}
+                                    </Text>
+                                    <Text numberOfLines={1}>{pronouns}</Text>
+                                </View>
+                            ) : null}
+                            {shouldShowLocalTime && <AutoUpdateTime timezone={timezone} />}
+                        </View>
+                        {!isCurrentUser && Boolean(login) && (
+                            <MenuItem
+                                title={`${props.translate('common.message')}${displayName}`}
+                                icon={Expensicons.ChatBubble}
+                                onPress={() => Report.navigateToAndOpenReport([login])}
+                                wrapperStyle={styles.breakAll}
+                                shouldShowRightIcon
+                            />
+                        )}
+                    </ScrollView>
+                )}
+                {!hasMinimumDetails && isLoading && <FullScreenLoadingIndicator style={styles.flex1} />}
+                {!hasMinimumDetails && !isLoading && (
```

Comment:
Since this is the case where the API has returned but returned nothing (we'll return the default avatar usually, but maybe this will show if an error occurs) - how about we make a new variable like `shouldShowBlockingView` and have a comment explaining that we'll only really do that if we're in an error state?